src/network/http/websocket.cpp

```diff
@@ -173,6 +154,14 @@ namespace fc { namespace http {
               return _ws_connection->get_request_header(key);
             }
 
+            virtual std::string get_host()override
+            {
+               auto header = get_request_header("BS-Forwarded");
+               if (header.length() > 2)
```

Comment:
why 2?

Reply:
Because I love magic numbers. Perhaps you don't like 2, so I replaced it with 3. (actually replaced with !string::empty() )